mmv1/products/dlp/api.yaml

```diff
@@ -309,6 +309,18 @@ objects:
                       - :TOP
                       - :RANDOM_START
                     default_value: :TOP
+                  - !ruby/object:Api::Type::Array
+                    name: 'identifyingFields'
+                    description: |
+                      Specifies the BigQuery fields that will be returned with findings (each field is expected to be comma separated).
```

Comment:
I'm not sure about the `(each field is expected to be comma separated)` part of the description here. What is this referring to? It seems like it could mislead a Terraform user into thinking they need to specify their config with a comma-separated string.

Reply:
Done.